docs/source/changes.rst

```diff
@@ -5,6 +5,47 @@
 Changes
 =======
 
+v0.5.0
+======
+
+**Breaking change**
+
+* Remove the ``BoxCox`` transform. ( `#222`_ )
```

Comment:
Maybe changed to:
* Remove the ``BoxCox`` transform, and include support for yeo-johnson and box-cox transformation through ``sklearn.preprocessing.PowerTransformer``. ( `#222`_ )

Reply:
Thanks! That's exactly what I want. Will replace all the short version with yours

Reply:
I follow your decision. I kind of think of this as an add-on to the breaking change.